test/models/imports/podcast_rss_import_test.rb

```diff
@@ -134,23 +144,23 @@
         "-4b1c-bbc6-2912d79d014f/248-Atom-in-the-Garden-of-Eden.mp3"
     end
     let(:sample_link3) do
-      "http://media.blubrry.com/some_name/www.podtrac.com/pts/redirect.mp3/blah"
+      "https://pts.podtrac.com/redirect.mp3/pdst.fm/e/chtbl.com/track/7E7E1F/blah"
     end
 
     it "can make a good guess for an enclosure prefix" do
       item = feed.entries.first
-      _(importer.enclosure_prefix(item)).must_equal "https://dts.podtrac.com/redirect" \
-        ".mp3/media.blubrry.com/transistor/"
+      _(importer.enclosure_prefix(item)).must_equal "https://dts.podtrac.com/redirect.mp3/media.blubrry.com/transistor/"
```

Comment:
i think i'm still not sure where this prefix comes from. Transistor is the podcast name, and so it's guessing the enclosure by using the podcast title?

Reply:
i guess the prefix is everything in the sample links up to `redirect.mp3`?

Reply:
and then what follows is a redirect url?

Reply:
These "prefixes" vary a bit by host - some of them are longer/shorter than others.  But when you request them, you get a bunch of redirects - here's a more realistic example:

```
GET https://pts.podtrac.com/redirect.mp3/pdst.fm/e/chtbl.com/track/7E7E1F/dovetail.prxu.org/123/abc/file.mp3
302 -> https://pdst.fm/e/chtbl.com/track/7E7E1F/dovetail.prxu.org/123/abc/file.mp3
302 -> https://chtbl.com/track/7E7E1F/dovetail.prxu.org/123/abc/file.mp3
302 -> https://dovetail.prxu.org/123/abc/file.mp3
302 -> https://dovetail-cdn.prxu.org/123/abc/def/file.mp3
200 -> finally we get to download the file!
```

Along each of those, the redirect hosts are tracking something about the request.  Until we finally get to dovetail.

Since Feeder only knows the `dovetail.prxu.org` URL, we need to save the rest of the chain in the `feed.enclosure_prefix` field as:

```
https://pts.podtrac.com/redirect.mp3/pdst.fm/e/chtbl.com/track/7E7E1F/
```

So we can add that to the beginning of every dovetail mp3 url in the feed.